developers/testing.md

```diff
@@ -188,11 +188,11 @@ One thing to note is that while the normal rake test command will migrate the da
 not (in the interest of saving time). To do this, simply run this in your foreman directory:
 
 ```
-RAILS_ENV=test rake db:drop && RAILS_ENV=test rake db:create && RAILS_ENV=test rake db:migrate
+RAILS_ENV=test rake db:drop && RAILS_ENV=test rake db:create && RAILS_ENV=test rake db:test:prepare && RAILS_ENV=test rake db:migrate
 ```
```

Comment:
not really answering your question, but I get a deprecation warning when I run db:prepare. Is there a way to do this that's more rails4-like?